README.md

```diff
@@ -52,24 +51,6 @@ the top in a command-prompt (here assuming Windows and git bash):
 After this `venv/Scripts/odg.exe` will exist and can be called
 from anywhere to run it.
 
-
```

Comment:
I don't think this PR should contain any py2.7 removal. Let's do that in a separate cleanup-commit.

Reply:
Commit has been reverted (https://github.com/Laerdal/python-objdictgen/pull/2/commits/8afaef5193153b0469732214d348bace8c6f546b) and cherry-picked onto separate branch (https://github.com/chrisinabox/python-objdictgen/tree/cleanup-remove-references-to-py2)